site/en/docs/extensions/whatsnew/index.md

```diff
@@ -18,56 +18,57 @@ and related policy or other changes.
 
 ### Chrome 100: `i18n.getMessage()` support in extension service workers {: #m100-i18n-getmessage }
 
+January 27, 2022
+
 The [`chrome.i18n.getMessage()`](/docs/extensions/reference/i18n/#method-getMessage) API is now
-supported in extension service worker contexts. This change landed in Canary on January 27, 2022
-and is expected to reach Stable in Chrome 100.
+supported in extension service worker contexts.
+
+### Chrome 99: `match_origin_as_fallback` in Canary {: #canary-match-origin-as-fallback }
 
-### 2022.01.05: `match_origin_as_fallback` in Canary {: #canary-match-origin-as-fallback }
+January 5, 2022
 
 Content scripts can now specify the `match_origin_as_fallback` key to inject into frames that are
 related to a matching frame, including frames with `about:`, `data:`, `blob:`, and `filesystem:`
 URLs.  See the [content scripts](/docs/extensions/mv3/content_scripts/#injecting-in-related-frames)
 documentation for details.
 
-### 2021.12.30: Extension service worker support for `file:` schemes in Canary {: #canary-file-access }
+### Chrome 99: Extension service worker support for `file:` schemes in Canary {: #canary-file-access }
```

Comment:
```suggestion
### Chrome 99: extension service worker support for `file:` schemes in Canary {: #canary-file-access }
```

Reply:
This is per the Chicago Manual of Style, which is a Google standard. You probably don't have a copy. Since we use sentence case for titles, the applicable rule is 6.62 in the 17th edition.

Please make sure all the headings in this document conform. Some of them aren't covered by this PR.

Reply:
I didn't realize we followed Chicago, thanks for calling that out. I'll use this PR to make all titles sentence case.